frontend/src/components/workflows/SubmissionFormRawProjections.tsx

```diff
@@ -12,7 +12,7 @@ import { SubmissionFormSharedFragment$key } from "./__generated__/SubmissionForm
 import { JSONObject } from "../../types";
 import { useState } from "react";
 import WorkflowStatus from "./WorkflowStatus";
-import { WorkflowStatusQuery$data } from "./__generated__/WorkflowStatusQuery.graphql";
+import { WorkflowSubscriptionHandlerSubscription$data } from "./__generated__/WorkflowSubscriptionHandlerSubscription.graphql";
```

Comment:
I think this import needs to be changed to import `WorkflowStatusSubscription$data`, is that correct?